server_addon/houdini/client/ayon_houdini/plugins/load/load_hda.py

```diff
@@ -71,4 +93,45 @@ def update(self, container, context):
 
     def remove(self, container):
         node = container["node"]
+        parent = node.parent()
         node.destroy()
+
+        if parent.type().category() == hou.objNodeTypeCategory():
+            return
```

Comment:
What does this solve? :) When would this happen?

Reply:
This line skip removing the avalon container itself.
this will happen when removing a loaded HDA of type Object while there are no loaded products. 
Therefore, The parent of the Obj HDA will be the Avalon container itself.

Reply:
Can't we be more explicit and just skip deletion if it's explicitly that particular case? Like if `parent.path() == lib.AVALON_CONTAINER`?

Reply:
Thank you https://github.com/ynput/ayon-core/pull/456/commits/320f97c9c68ff5acd220c5da8f632258f935c2dd